### PR TITLE
This updates PR 6 with the new dependency: pyinstrument_cext.

### PR DESCRIPTION
**Pull request by moorepants**

### CODE REVIEW

**Comment by moorepants** [comment]
This is ready to merge.

**Comment by moorepants** [comment]
@conda-forge/help-python Would someone mind merging this?

**Comment by CJ-Wright** [comment]
@conda-forge/pyinstrument

**Comment by moorepants** [comment]
@isuruf Would you mind merging this?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {% set name = "pyinstrument" %}
-{% set version = "0.13.2" %}
+{% set version = "2.0.2" %}
 {% set bundle = "tar.gz" %}
 {% set hash_type = "sha256" %}
-{% set hash_val = "a0a645d69df12551214311fdf2e454c12b96cb0abc0f5dcc3067d8ef71713e02" %}
+{% set hash_val = "c04f3a0771185c66db7acb92db518be3db79253aad790f7c81da57d5ed9253bd" %}
 {% set build = 0 %}
```
Comment (isuruf) [inline recipe/meta.yaml]: Can you remove this line since `build` is set in Line 21?

Comment (moorepants) [reply recipe/meta.yaml]: Done.

**Pull request merged**